docs/specs/tripgo.swagger.yaml

```diff
@@ -2172,6 +2190,42 @@ definitions:
       #     - label: Sat - Sun
       #       price: 22
 
+  Restriction:
+    description: Information about the car park restriction.
+    properties:
+      type:
+        type: string
+        enum:
+          - MAX_STAY
+          - NO_RESTRICTION_WHEN_CLOSED
+          - ALLOWED_ONLY
+          - NOT_ALLOWED
+      value:
+        description: depending on the restriction type
+        type: string | boolean | integer
+
+  Entrance:
+    description: Information about the car park entrance.
+    properties:
+      type:
+        description: Entrance type identifier
+        type: string
+        enum:
+            - ENTRANCEEXIT
+            - ENTRANCEONLY
+            - EXITONLY
+            - PEDESTRIAN
+            - DISABLEDPEDESTRIAN
+            - PERMIT
```

Comment:
Are these mutually exclusive? Also would be better to separate words with underscores in the enum names.

Reply:
> Are these mutually exclusive? 

Yes.
>Also would be better to separate words with underscores in the enum names.

Done.